tests/email-validator.spec.ts

```diff
@@ -0,0 +1,54 @@
+import test, { expect } from "@playwright/test";
+
+const valid_emails = [
+  "email@test.com",
+  "email@test.xyz",
+  "email@test.a",
+  "example.address@t.co",
+  "example.address@a.b.c.d.e.f.g",
+  "example_address@a.b.c",
+  "example_address_2@a.b.c",
+];
+
+for (const address of valid_emails) {
```

Comment:
Would it be possible to nest these tests into a single one? I believe Playwright's error messages already suffice.